apps/sms/test/marionette/new_activity_test.js

```diff
@@ -102,7 +102,7 @@ marionette('Messages as "new" activity target', function() {
         return !!window.wrappedJSObject.Compose;
       }), 'XSS should not be performed');
 
-      assert.isTrue(messagesApp.Composer.sendButton.enabled());
+      client.waitFor(() => messagesApp.Composer.sendButton.enabled());
```

Comment:
> If it still not enough, maybe we could add a class to composer to represent composer status is ready?

No, it's still not enough, when we open Composer with a draft (e.g. in case when app is opened as activity), ```updateSendButton``` method inside Compose.js is called 4/5 times asynchronously:
* beforeEnterComposer ->  Compose.clear() -> ....;
* afterEnterComposer -> handleDraft() -> Compose.clear() -> ...;
* afterEnterComposer -> handleDraft() -> Compose.append() -> ...;
* toFieldInput() (throttled, in FxDesktop called twice) -> Compose.refresh() -> ...

> Not quite sure about the timing of this `client.waitFor`, maybe we can wait for specific element in `launchAsActivity`?

Mmm, honestly I don't have idea what else we can wait without refactoring current Compose, do you have idea?